mungegithub/github/github.go

```diff
@@ -1800,46 +1852,47 @@ func (config *Config) ForEachIssueDo(fn MungeFunction) error {
 			Labels:      config.Labels,
 			Direction:   "asc",
 			ListOptions: github.ListOptions{PerPage: 100, Page: page},
+			Since:       config.since,
 		}
 		issues, response, err := config.client.Issues.ListByRepo(config.Org, config.Project, listOpts)
 		config.analytics.ListIssues.Call(config, response)
 		if err != nil {
 			return err
 		}
 		for i := range issues {
-			issue := issues[i]
-			if issue.Number == nil {
-				glog.Infof("Skipping issue with no number, very strange")
-				continue
-			}
-			if issue.User == nil || issue.User.Login == nil {
-				glog.V(2).Infof("Skipping PR %d with no user info %#v.", *issue.Number, issue.User)
-				continue
-			}
-			if *issue.Number < config.MinPRNumber {
-				glog.V(6).Infof("Dropping %d < %d", *issue.Number, config.MinPRNumber)
-				continue
-			}
-			if *issue.Number > config.MaxPRNumber {
-				glog.V(6).Infof("Dropping %d > %d", *issue.Number, config.MaxPRNumber)
-				continue
-			}
-			glog.V(2).Infof("----==== %d ====----", *issue.Number)
-			glog.V(8).Infof("Issue %d labels: %v isPR: %v", *issue.Number, issue.Labels, issue.PullRequestLinks != nil)
-			obj := MungeObject{
+			obj := &MungeObject{
 				config:      config,
-				Issue:       issue,
+				Issue:       issues[i],
 				Annotations: map[string]string{},
 			}
-			if err := fn(&obj); err != nil {
-				continue
+			err := config.runMungeFunction(obj, fn)
```

Comment:
there are mungers that need to run not just when things change -- like the stale issue nagging. How will this be handled?